app/src/main/res/values/strings.xml

```diff
@@ -1628,6 +1628,14 @@ Partially means that a wheelchair can enter and use the restroom, but no handrai
 
     <string name="overlay_street_parking">Street parking</string>
 
+    <!-- Used for surfaces unknown by SC. Does not mean it is invalid. The wording that it is "unknown" is avoided so that the user does not think an action is necessary from him -->
```

Comment:
Don't assume the user is male, see e.g. https://langeek.co/en/grammar/course/336/them

```suggestion
    <!-- Used for surfaces unknown by SC. Does not mean it is invalid. The wording that it is "unknown" is avoided so that the user does not think an action is necessary from them -->
```